nifi-commons/nifi-site-to-site-client/src/main/java/org/apache/nifi/remote/client/SiteToSiteClient.java

```diff
@@ -560,7 +560,8 @@ public SiteToSiteClient build() {
         }
 
         /**
-         * @return the configured URL for the remote NiFi instance
+         * @return the configured URL for the remote NiFi instance.
+         * The URL string possibly contains multiple URL entries in comma-separated format.
          */
         public String getUrl() {
```

Comment:
SiteToSiteReportingTask has its own UI and it doesn't allow URL in a comma-separated format. We need to update the reporting task, too. However, since there're other features such as HTTP transport and Proxy support to add to the reporting task, I issued a separate JIRA [SiteToSiteProvenanceReportingTask to support Site-to-Site recently added features](https://issues.apache.org/jira/browse/NIFI-3030).